.github/workflows/generator_container_slsa3.yml

```diff
@@ -34,6 +34,12 @@ on:
         description: "Password to log in the container registry."
       image:
         description: "The OCI image name. This must not include a tag or digest."
+      provenance-registry-username:
+        description: "Username when publishing to provenance registry instead of image registry"
```

Comment:
nit: `Username when publishing to provenance registry (option 'prove') instead of image registry`

Reply:
done